src/axom/inlet/Table.hpp

```diff
@@ -238,8 +250,8 @@ class Table : public SchemaCreator, public std::enable_shared_from_this<Table>
    * \return Shared pointer to the created Field
    *****************************************************************************
    */
-  std::shared_ptr<Table> addBoolArray(const std::string& name,
-                                      const std::string& description = "");
+  std::shared_ptr<Verifiable> addBoolArray(const std::string& name,
```

Comment:
Ok, so you can still chain calls like `->required(true)`, but you no longer get the field.

I suppose you can no longer return a `Table` or `Field` since an array is `Verifiable` while a `Field` is a `VerifiableScalar` ?

Reply:
It has to do with the `Aggregate` types - the call to `addBoolArray` could either add a single array (returns Table) or, if the calling table is an array of structs, an array for each struct in the array of structs (returns AggregateTable).   I think adding fields to an array of booleans is invalid anyways, but in the general case it does restrict the user a bit.